extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -243,7 +243,7 @@ export abstract class AzureAuth implements vscode.Disposable {
 	 * @param resource
 	 * @param refreshToken
 	 */
-	public async refreshToken(tenant: Tenant, resource: Resource, refreshToken: RefreshToken | undefined): Promise<OAuthTokenResponse> {
+	public async refreshToken(tenant: Tenant, resource: Resource, refreshToken: RefreshToken | undefined): Promise<OAuthTokenResponse> | undefined {
```

Comment:
I still don't see anything explaining how/why this would return undefined. Please look through this and add the relevant description to the doc comments here so people looking at this know what to expect.

Reply:
Added documentation for undefined.

![image](https://user-images.githubusercontent.com/87730006/166081337-93801a19-9928-4dd9-98fc-e9a7f052148d.png)